### ISSUE TITLE
Automate updating of publication lists and bar graphs

### ISSUE THREAD

**Issue by ckhroulev** [pism/pism.github.io#4]
See `publications/Makefile` and https://github.com/pism/misc/blob/master/pism-publications.py

**Comment by ckhroulev** [comment]
Done.

**Comment by bueler** [comment]
I pushed a change to `publications/applications.bib` with the expectation that `https://pism.github.io/publications/` would be updated.  No luck after an hour.  Is everybody just accepting of totally non-responsive web updates in the modern world?  Alternatively (and probably), auto-generation requires some additional action on my part?  Looking at `.circleci/config.yml` and `Makefile` I figured that the CircleCi webhook on this repo should trigger a `make` (which catches the `update` target) after my push, but probably I don't understand.
